app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,28 @@
+
+<div class="col-lg-5">
+  <div class="card">
+    <div class="card-header">
+     <h2 class="text-center"> Login </h2>
+    </div>
+    <div class="card-body">
+      <%= bootstrap_form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+        <div class="field">
+          <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+        </div>
+        <div class="field">
+          <%= f.password_field :password, autocomplete: "current-password" %>
+        </div>
+        <% if devise_mapping.rememberable? %>
+          <div class="field">
+            <%= f.check_box :remember_me %>
+          </div>
+        <% end %>
+        <div class="actions d-flex justify-content-center mt-4">
+          <%= f.submit "Log in" , class: 'btn btn-primary col-lg-3' %>
+        </div>
+      <% end %>
+      <%= render "devise/shared/links" %>
+    </div>
+  </div>
+</div>
```

Comment:
Try to improve your HTML using different types of tags and fewer divs, for a semantic web.

Reply:
https://dev.to/kenbellows/stop-using-so-many-divs-an-intro-to-semantic-html-3i9i

Reply:
Yes i recomend to use semantic HTML ! to improve sintaxis